src/services/dashboards/resource.js

```diff
@@ -383,6 +384,7 @@ export async function resourceData(scopes, skipResources = false, skipTopics = f
           {
             calculatedStatus: REPORT_STATUSES.APPROVED,
             startDate: { [Op.ne]: null },
+            createdAt: { [Op.gt]: '2022-12-01' },
```

Comment:
I think it would be wise to store this date as a variable and use that throughout.

Reply:
done.